### ISSUE TITLE
Pin confusion on lock screen

### ISSUE THREAD

**Issue by yassine-cc** [Genymobile/scrcpy#3909]
- [ x] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [ x] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Linux
 - scrcpy version: 1.21
 - installation method: apt package manager
 - device model: Google Pixel 6
 - Android version: 13 (Graphene OS)

**Describe the bug**
I don't know if this is a bug caused by scrcpy or by android, but before 2 or 3 months ago, when on lock screen, i used to just type the password/pin with my keyboard and press enter to login (On the actual lock screen and not on the password window)

However when i do this now, the password is being evaluated as the sim card PIN ![image](https://user-images.githubusercontent.com/47945562/231237340-f8b25b3f-98c3-4193-ba1a-c22546f0356a.png)

So basically to unlock the phone now, i need to trigger the lock screen, swipe up for the password screen (not mirrored) and then type the pin and enter.

**Comment by rom1v** [comment]
Probably an Android upgrade which changed the behavior.
